test/CQRS/LeanCode.CQRS.AspNetCore.Tests/Middleware/CQRSValidationMiddlewareTests.cs

```diff
@@ -36,6 +40,11 @@ public CQRSValidationMiddlewareTests()
     protected override void ConfigureServices(IServiceCollection services)
     {
         services.AddSingleton(_ => validatorResolver);
+        services.AddLogging(logging =>
+        {
+            logging.AddProvider(NullLoggerProvider.Instance);
```

Comment:
Shouldn't this be handled by the `TryAddNullLeanCodeLogger`? method?

Reply:
Or even shouldn't this be omitted, as we are supposed to use our `ILogger<T>`?

Reply:
I think omitted would be the way to go, but I wasn't sure if our middlewares happen to pull in other classes that use classic .NET logger (or not). I'll remove it if it isn't needed.